crate/kmip/src/kmip/kmip_operations.rs

```diff
@@ -1080,6 +1085,19 @@ pub struct ReKeyKeyPairResponse {
     pub public_key_unique_identifier: UniqueIdentifier,
 }
 
+#[derive(Debug, Serialize, Deserialize, Default)]
+#[serde(rename_all = "PascalCase")]
+pub enum ReKeyKeyPairAction {
```

Comment:
Why not `RekeyKeyPairAction` (it seems to rekey [is a thing](https://wikiless.tiekoetter.com/wiki/Rekeying_(cryptography)?lang=en))?
```suggestion
pub enum RekeyKeyPairAction {
```

Reply:
I used the same naming convention as the other structs